c/polyjuice.h

```diff
@@ -820,23 +785,20 @@ int load_globals(gw_context_t* ctx, uint32_t to_id) {
     debug_print_data("invalid to account script args", raw_args_seg.ptr, raw_args_seg.size);
     return FATAL_POLYJUICE;
   }
-  /** read g_compatible_chain_id from Godwoken RollupConfig */
+  /** read g_chain_id from Godwoken RollupConfig */
   mol_seg_t rollup_config_seg;
   rollup_config_seg.ptr = ctx->rollup_config;
   rollup_config_seg.size = ctx->rollup_config_size;
-  mol_seg_t id_u32_seg = MolReader_RollupConfig_get_compatible_chain_id(&rollup_config_seg);
-  memcpy(&g_compatible_chain_id, id_u32_seg.ptr, id_u32_seg.size);
-  debug_print_int("compatible_chain_id", g_compatible_chain_id);
+  mol_seg_t id_u64_seg = MolReader_RollupConfig_get_chain_id(&rollup_config_seg);
+  memcpy(&g_chain_id, id_u64_seg.ptr, id_u64_seg.size);
+  debug_print_int("chain_id", g_chain_id);
   debug_print_int("creator_account_id", g_creator_account_id);
```

Comment:
Should we assert `g_creator_account_id == GW_DEFAULT_ETH_REGISTRY_ACCOUNT_ID`  here?

Reply:
I think they have different roles. Shouldn't be the same.